packages/jaspr_router/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: jaspr_router
 description: A routing component for jaspr.
-version: 0.2.3
+version: 0.2.4
```

```suggestion
version: 0.2.3
```

Will be increased by the release pipeline.